src/config/index.ts

```diff
@@ -0,0 +1,19 @@
+import dotenv from 'dotenv';
+import path from 'path';
+
+dotenv.config({ path: path.join(__dirname, '/../../.env') });
```

Comment:
путь лучше указывать не слешами, а отдельными элементами через path.join
```suggestion
dotenv.config({ path: path.join(__dirname, '..', '..', '.env') });
```

Reply:
Почему?